src/locales/en-US/commands.json

```diff
@@ -1,182 +1,190 @@
+    "noSentence": "You need to give me a sentence to flip"
+  },
+  "npm": {
+    "commandDescription": "Shows information about a NPM package",
+    "commandUsage": "<package name>",
+    "noDescription": "No description provided",
+    "notFound": "Package not found.",
+    "noNameProvided": "You need to give me a package name."
+  },
+  "poll": {
+    "commandDescription": "Creates a poll",
+    "commandUsage": "<poll title> [|option 1|option 2|option 3|...]",
+    "noQuestion": "You need to give me something to be voted",
+    "tooManyOptions": "You can't exceed {{maxOptions}} options"
+  },
+  "nowPlaying": {
+    "commandDescription": "Shows the name of the song being played"
+  },
+  "pause": {
+    "commandDescription": "Pauses or resumes the music playback"
+  },
+  "play": {
+    "commandDescription": "Plays a certain song or adds it to the queue",
+    "commandUsage": "<track name or URL>",
+    "noTrackIdentifier": "You need to give me a track name or URL!"
+  },
+  "queue": {
+    "commandDescription": "Shows the current queue"
+  },
+  "skip": {
+    "commandDescription": "Skips the current song"
+  },
+  "stop": {
+    "commandDescription": "Stops the music playback"
+  },
+  "volume": {
+    "commandDescription": "Adjusts the volume of the music playback",
+    "commandUsage": "<0-150>",
+    "missingVolumeParameter": "You need to give me the volume level!",
+    "invalidVolumeParameter": "You need to give me a valid number!",
+    "volumeSet": "Volume set to {{volume}}%"
+  },
+  "help": {
+    "commandDescription": "Shows information about commands or lists them",
+    "commandUsage": "[command name]",
+    "listTitle": "Available commands",
+    "prefix": "Prefix",
+    "youCanUse": "You can also use {{- botMention}} as a prefix",
+    "specificInformation": "Type {{helpString}} for command specific information",
+    "aliases": "Aliases",
+    "noDescriptionProvided": "No description provided",
+    "noUsageProvided": "No usage provided",
+    "commandNotFound": "I couldn't find a command with that name"
+  },
+  "setlanguage": {
+    "commandDescription": "Changes the bot's language",
+    "commandUsage": "<language code>",
+    "noCode": "You need to give me a language code.",
+    "availableLanguages": "Available languages",
+    "changedSuccessfully": "Language changed successfully to {{lang}}!"
+  },
+  "e621": {
+    "hereIsYour_yiff": "Here's your yiff!"
+  },
+  "pay": {
+    "commandDescription": "Donates $t(commons:currency_plural) to a user",
+    "commandUsage": "<@member> <value>",
+    "noMember": "You need to give me a valid member mention",
+    "noValue": "You need to give me a valid number of $t(commons:currency_plural) to send",
+    "notEnoughMoney": "You don't have enough $t(commons:currency_plural) to perform this transaction!",
+    "transactionSuccessful": "Transaction completed with success! {{- receiver}} has received **{{- value}} $t(commons:currency_plural)**!"
```

Comment:
No need to escape placeholders with `-` anymore, as we have disabled escaping globally.